project/plugins.sbt

```diff
@@ -6,4 +6,6 @@ addSbtPlugin("org.scalameta"      % "sbt-mdoc"       % "2.2.16")
 addSbtPlugin("ch.epfl.scala"      % "sbt-bloop"      % "1.4.6")
 addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"     % "0.4.3")
 addSbtPlugin("com.geirsson"       % "sbt-ci-release" % "1.5.5")
-addSbtPlugin("ch.epfl.scala"      % "sbt-scalafix"   % "0.9.34")
+addSbtPlugin("dev.zio"            % "zio-sbt-website" % "0.1.5+28-4b5b0374-SNAPSHOT")
```

Comment:
May I ask you to use latest version of `zio-sbt-website`, currently `0.3.4`?

Reply:
so I tried this and got a build failure - maybe the binary has not been published?

Reply:
I saw that you used it correctly but it needs some migration that is not documented.